docs/user-research/MO-user-groups-feedback.md

```diff
@@ -0,0 +1,276 @@
+
+* AD: We have set up a feedback page under the retooling project pages & we'll
+  keep putting recommendations on there [We said thanks.].
+
+* MB: In general, Cylc meets our needs, but Rose does not. [We said this was
+  noted.]
+
+
+#### From feedback pages
+
+Notable points (relating to the Cylc UI) from feedback pages:
+
+* [From the agenda notes:] "TSUG noted that this [the Rose replacement] will
+  be of most benefit to users represented here. The *Cylc upgrades are nice
+  to have* but the Rose upgrades will be vital to allow us to get the most
+  out of our trials."
+
+NB: those pages contain quite a lot of useful feedback, but not much
+(just the above) relating to the Cylc UI, mainly to Rose config-edit, plus one
+non-UI Cylc matter (which is of relevance to future Python API work):
+
+* Under the section 'User requirements for new suite control system', from MB:
+  "Better support for determining more complex aspects of the graph of a
+  suite. Some aspects of the suite graph of an ideal version of an evaluation
+  suite would be sufficiently complex that using jinja2 is unwieldy, limiting
+  and difficult to maintain. It would be useful to have the ability to define
+  the suite programmatically either as an initial app, or by some other means.
+
+
+### CRUG (Climate/Coupled Rose User Group)
+
+#### From the meeting
+
+* DC: For the Tree View, will we still be able to see [task-]job times? [We
+  said that they are not shown on the mock-up but will be accessible, yes,
+  though the Tree View may be more lightweight such that they (as with other
+  column data) are only displayed when requested, rather than by default.]
+
+* JCR: It would be useful if there were tools to analyse the performance of a
+  workflow, e.g. a Gantt Chart. [We mentioned our plan to have a Gantt View,
+  though said probably only in later versions of Cylc 8, along with other
+  components to provide analytics for workflows & their individual tasks.]
+
+* JR: Dot View is not on the mock-up, will it still be there in the new UI?
+  I think Dot View is the most useful, for example to get access to the jobs &
+  times just from the tasks. [We said this is noted & useful to know.]
+* CM: Yes [agreeing with JR previous comment] I find Dot View useful to
+  determine "flow" through suites. [Ditto.]
+
+* RG: Will we be able to customise the datetime formats? I find them very
+  unwieldy in their full form. [We explained that it would be difficult &
+  impractical to change the datetimes used internally.]
+* RG: (clarifying previous comment) I don't mean customising them as used &
+  processed for the suite internals, just for the UI display, i.e. what we
+  will see on the front-end. [We stated that we have considered datetime
+  customisation on the UI, & hope to enable some means to make the datetime
+  formats clearer there, for example by including the ':' & '-' delimiters even
+  if not used in the configuration, to break apart the datetime components.]
```

Comment:
Presumably you mentioned (e.g.) `cycle point format = %Y-%m-%dT%H:%M`?

Reply:
I can't say for certain, since it's not in my notes & my memory is fading & the meetings were a month ago now, but I suspect it was probably mentioned. Regardless, I am confident the person in question would have been aware of that configuration item, &/or as a Rose user likely know about ``rose date``.

Reply:
> Presumably you mentioned (e.g.) cycle point format = %Y-%m-%dT%H:%M?

Yes.

Reply:
So do we have this point covered (users can set the date-time format as they like) or do they want us to display formatted date times in the UI regardless of the internal format used?  That may not be a good idea as the date times displayed in the UI won't match those passed to task jobs and used in filenames etc.